src/meta/src/barrier/checkpoint/recovery.rs

```diff
@@ -478,8 +486,12 @@ impl DatabaseStatusAction<'_, EnterInitializing> {
 pub(crate) struct EnterRunning;
 
 impl DatabaseStatusAction<'_, EnterRunning> {
-    pub(crate) fn enter(self) {
+    pub(crate) fn enter(self, control_stream_manager: &ControlStreamManager) {
```

Comment:
Looks dirty to pass the `control_stream_manager ` just for accessing the event log manager. We can store the `env` in `CheckpointControl` and then we can access the `env` from `self`. And also in `DatabaseStatusAction<'_, EnterReset> ` we can use `self` to access the `env`.